apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf

```diff
@@ -340,6 +340,28 @@ Commands can support following wildcards:\n
     }
   }
 
+  jwt {
```

Comment:
We should also add Zh translation.

Reply:
Let me add it in another PR?

Reply:
see: https://github.com/emqx/emqx/pull/7769